src/components/ChallengeReaction.vue

```diff
@@ -1,54 +1,64 @@
 <template>
-    <v-container class="reaction" :class="{ 'chosen-reaction': props.challengeInput.isChosenAnswer }">
-        <v-row v-if="props.challengeInput.isChosenAnswer">
-            <v-icon class="ml-2 my-2">mdi-bookmark</v-icon> 
-            <p class="d-flex align-center">Het gekozen antwoord door : {{ 'Company naam' }}</p>
-            <v-divider></v-divider>
-        </v-row>
+  <v-container
+    class="reaction"
+    :class="{ 'chosen-reaction': props.challengeInput.isChosenAnswer }"
```

Comment:
Waarom staat er code voor het plaatsen van reacties in deze pull request?

Reply:
Goed dat je dit ziet, hoort er idd niet bij